### PR TITLE
Views and css

### PR DESCRIPTION
**Pull request by RusuSebastian**

### CODE REVIEW

File: UserOperation.Web/Areas/Identity/Pages/Account/Login.cshtml [hunk 1]
```diff
@@ -35,32 +35,32 @@
                     <button id="login-submit" type="submit" class="w-100 btn btn-lg btn-primary">Log in</button>
                 </div>
                 <div>
-                    <p>
+                    @*<p>
```
Comment (ana-maria-apopei) [inline UserOperation.Web/Areas/Identity/Pages/Account/Login.cshtml:38]: Remove commented code

File: UserOperation.Web/Areas/Identity/Pages/Account/Login.cshtml [hunk 2]
```diff
@@ -35,32 +35,32 @@
...
                         <a id="forgot-password" asp-page="./ForgotPassword">Forgot your password?</a>
-                    </p>
+                    </p>*@
```
Comment (ana-maria-apopei) [inline UserOperation.Web/Areas/Identity/Pages/Account/Login.cshtml:40]: Remove commented code

File: UserOperation.Web/Areas/Identity/Pages/Account/Login.cshtml [hunk 3]
```diff
@@ -35,32 +35,32 @@
...
                     <p>
                         <a asp-page="./Register" asp-route-returnUrl="@Model.ReturnUrl">Register as a new user</a>
                     </p>
-                    <p>
+                    @*<p>
```
Comment (ana-maria-apopei) [inline UserOperation.Web/Areas/Identity/Pages/Account/Login.cshtml:44]: Remove commented code

File: UserOperation.Web/Areas/Identity/Pages/Account/Login.cshtml [hunk 4]
```diff
@@ -35,32 +35,32 @@
...
                         <a id="resend-confirmation" asp-page="./ResendEmailConfirmation">Resend email confirmation</a>
-                    </p>
+                    </p>*@
                 </div>
             </form>
         </section>
     </div>
-    <div class="col-md-6 col-md-offset-2">
+    @*<div class="col-md-6 col-md-offset-2">
```
Comment (ana-maria-apopei) [inline UserOperation.Web/Areas/Identity/Pages/Account/Login.cshtml:51]: Remove commented code

File: UserOperation.Web/Areas/Identity/Pages/Account/Login.cshtml [hunk 5]
```diff
@@ -35,32 +35,32 @@
...
             @{
                 if ((Model.ExternalLogins?.Count ?? 0) == 0)
                 {
-                    <div>
-                        <p>
-                            There are no external authentication services configured. See this <a href="https://go.microsoft.com/fwlink/?LinkID=532715">article
-                            about setting up this ASP.NET application to support logging in via external services</a>.
-                        </p>
-                    </div>
+                    //<div>
```
Comment (ana-maria-apopei) [inline UserOperation.Web/Areas/Identity/Pages/Account/Login.cshtml:58]: Remove commented code

File: UserOperation.Web/Areas/Identity/Pages/Account/Register.cshtml [hunk 6]
```diff
@@ -4,12 +4,12 @@
     ViewData["Title"] = "Register";
 }
 
-<h1>@ViewData["Title"]</h1>
+<h1 style="text-align:center">@ViewData["Title"]</h1>
```
Comment (ana-maria-apopei) [inline UserOperation.Web/Areas/Identity/Pages/Account/Register.cshtml:7]: Replace inline styling

File: UserOperation.Web/Areas/Identity/Pages/Account/Register.cshtml [hunk 7]
```diff
@@ -4,12 +4,12 @@
...
 
-<div class="row">
+<div class="row" style="display:flex;justify-content:center">
     <div class="col-md-4">
         <form id="registerForm" asp-route-returnUrl="@Model.ReturnUrl" method="post">
-            <h2>Create a new account.</h2>
+            <h2 style="text-align:center">Create a new account.</h2>
```
Comment (ana-maria-apopei) [inline UserOperation.Web/Areas/Identity/Pages/Account/Register.cshtml:12]: Replace inline styling

File: UserOperation.Web/Views/Shared/_FilesActionsSection.cshtml [hunk 8]
```diff
@@ -0,0 +1,17 @@
+﻿
+   <section class="files-btns " style="display:flex;align-items:center;justify-content:space-between" >
```
Comment (ana-maria-apopei) [inline UserOperation.Web/Views/Shared/_FilesActionsSection.cshtml:2]: Replace inline styling

**Review by ana-maria-apopei — CHANGES_REQUESTED** [review]

**Review by Emma-Sleghel — APPROVED** [review]

**Review by ana-maria-apopei — APPROVED** [review]

**Pull request merged**